hyper/compat.py

```diff
@@ -15,6 +15,12 @@
     # TODO log?
     ssl_compat = None
 
+try:
+    from urllib.parse import urlencode, urlsplit
+except ImportError:  # pragma: no cover
+    from urllib import urlencode
+    from urlparse import urlsplit
+
```

Comment:
Rather than putting this up here, can you put it in the relevant py2/py3 branches lower down? Saves the need for a try/except.